CorsixTH/Lua/graphics.lua

```diff
@@ -131,22 +135,23 @@ function Graphics:loadFontFile()
   elseif self.app.os == "macos" then
     os_path = "/Library/Fonts/Arial Unicode.ttf"
```

Comment:
No, why would we make it not work on macos? - actually why are we still overriding font in graphics.lua at all?